build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     }
     dependencies {
         //noinspection GradleDependency
-        classpath 'com.android.tools.build:gradle:3.5.3'
+        classpath 'com.android.tools.build:gradle:4.0.0'
```

Comment:
Any reason why you updated gradle version in this PR?

Reply:
Because of my Android Studio 4.1.